daemon/kmd/api/swagger.json

```diff
@@ -1345,9 +1345,9 @@
       "description": "APIV1POSTWalletRenameRequest is the request for `POST /v1/wallet/rename`",
       "type": "object",
       "properties": {
-        "wallet_id": {
+        "wallet_handle_token": {
           "type": "string",
-          "x-go-name": "WalletID"
+          "x-go-name": "WalletHandleToken"
```

Comment:
This is a [published KMD API](https://developer.algorand.org/docs/rest-apis/kmd/#post-v1walletrename) so I'd rather not change it at this time